.github/twitter.py

```diff
@@ -0,0 +1,53 @@
+import tweepy
+import requests
+import xml.etree.ElementTree as ET
+import os
+
+
+def main():
+    client = tweepy.Client(
+        consumer_key=os.environ("API_KEY"),
```

Comment:
this code won't run – os.environ is a dictionary so you want to do os.environ["API_KEY"] for example.